src/UniOneClient.php

```diff
@@ -128,20 +128,12 @@ public function httpRequest(string $path, array $body, array $headers = [], stri
             $body['message']['platform'] = 'phpsdk.'.self::VERSION;
         }
 
-        try {
-            // Send request.
-            $response = $this->httpClient->request($method, $path, [
-              'headers' => $requestHeaders,
-              'json' => $body,
-            ]);
-
-            return $response->getBody()->getContents();
-        } catch (BadResponseException $e) {
-            // handle exception or api errors.
-            return $e->getResponse()->getBody()->getContents();
-        } catch (TransferException $e) {
-            // handle exception or api errors.
-            return $e->getMessage();
-        }
+        // Send request.
+        $response = $this->httpClient->request($method, $path, [
+          'headers' => $requestHeaders,
+          'json' => $body,
+        ]);
+
+        return \json_decode($response->getBody()->getContents());
```

Comment:
We should send associative = true to always have an array. See: https://www.php.net/manual/en/function.json-decode.php